extensions/mapstore/src/test/java/com/hazelcast/mapstore/GenericMapStoreTest.java

```diff
@@ -611,6 +604,7 @@ private <K> GenericMapStore<K> createMapStore(Properties properties, HazelcastIn
 
         GenericMapStore<K> mapStore = new GenericMapStore<>();
         mapStore.init(instance, properties, mapName);
+        mapStore.awaitInitFinished();
```

Comment:
Some tests intentionally fail to create the mapping, so the `awaitInitFinished` throws an exception. See the failed tests.